### PR TITLE
fix(provider): add home flag to provider cli

### LINKED ISSUE #1141: Failed to query lease status
This issue is happening on mac os using the `os` keyring.
```
akash provider lease-status --dseq 4380 --from dep --provider akash1ccktptfkvdc67msasmesuy5m7gpc76z75kukpz --log_level info
panic: could not create directory "": mkdir : no such file or directory

goroutine 1 [running]:
github.com/tendermint/tendermint/config.EnsureRoot(0x0, 0x0)
    github.com/tendermint/tendermint@v0.34.8/config/toml.go:35 +0x305
github.com/cosmos/cosmos-sdk/server.interceptConfigs(0xc001167e60, 0xc0011e13c0, 0x4, 0xc000e5f608)
    github.com/cosmos/cosmos-sdk@v0.41.3/server/util.go:193 +0x914
github.com/cosmos/cosmos-sdk/server.InterceptConfigsPreRunHandler(0xc00112f8c0, 0x20, 0x93007d0)
    github.com/cosmos/cosmos-sdk@v0.41.3/server/util.go:126 +0x31a
github.com/ovrclk/akash/cmd/akash/cmd.NewRootCmd.func1(0xc00112f8c0, 0xc00124b780, 0x0, 0x8, 0x0, 0x0)
    github.com/ovrclk/akash/cmd/akash/cmd/root.go:82 +0x4f
github.com/tendermint/tendermint/libs/cli.concatCobraCmdFuncs.func1(0xc00112f8c0, 0xc00124b780, 0x0, 0x8, 0x0, 0x0)
    github.com/tendermint/tendermint@v0.34.8/libs/cli/setup.go:118 +0x7d
github.com/spf13/cobra.(*Command).execute(0xc00112f8c0, 0xc00124b700, 0x8, 0x8, 0xc00112f8c0, 0xc00124b700)
    github.com/spf13/cobra@v1.1.1/command.go:829 +0x587
github.com/spf13/cobra.(*Command).ExecuteC(0xc00112e580, 0x6501373, 0x5, 0xc000f65460)
    github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
    github.com/spf13/cobra@v1.1.1/command.go:895
github.com/spf13/cobra.(*Command).ExecuteContext(...)
    github.com/spf13/cobra@v1.1.1/command.go:888
github.com/ovrclk/akash/cmd/akash/cmd.Execute(0xc00112e580, 0x6acd8c0, 0xc00112ca20)
    github.com/ovrclk/akash/cmd/akash/cmd/root.go:116 +0x26d
main.main()
    github.com/ovrclk/akash/cmd/akash/main.go:14 +0x2a
```

Also of note: `v0.10.1` has a regression from `v0.10.0` where the `--log_level` flag's default isn't set properly and results in an error on all CLI commands.

### PR DESCRIPTION
**Pull request by troian**
fixes #1141

Signed-off-by: Artur Troian <troian.ap@gmail.com>

### CODE REVIEW

**Comment by hydrogen18** [comment]
On master this flag is already present when I do

File: provider/cmd/helpers.go [hunk 1]
```diff
@@ -22,10 +24,6 @@ func addCmdFlags(cmd *cobra.Command) {
 		panic(err.Error())
 	}
 
-	if err := cmd.MarkFlagRequired(flags.FlagHome); err != nil {
```
Comment (boz) [inline provider/cmd/helpers.go:25]: don't we still want this?

Comment (troian) [reply provider/cmd/helpers.go:25]: if it marked as required then it prompts to set `--home` and ignores default value from above

**Review by boz — APPROVED** [review]
:+1:

**Pull request merged**